evalai/urls.py

```diff
@@ -92,6 +92,7 @@
         schema_view.with_ui("redoc", cache_timeout=0),
         name="schema-redoc",
     ),
+    url(r"^api/", include("django_prometheus.urls")),
```

Comment:
@burnerlee I thought this would become `api/metrics`. Including prometheus urls at `api/` isn't a good idea. You can revert to `api/prometheus`

Reply:
@burnerlee @Ram81 I might be missing context here but what is the issue with using `api/metrics`

Reply:
@RishabhJain2018 I just wanted all prometheus under single root api path

Reply:
I see..okay. Sounds good!